creme/creme_config/forms/fields.py

```diff
@@ -113,8 +116,8 @@ def __init__(self, *, queryset, create_action_url='', user=None, **kwargs):
         self.creation_info(create_action_url, user)
 
 
-class CreatorEnumerableChoiceField(enum_forms.EnumerableChoiceField,
-                                   CreatorChoiceMixin):
+class CreatorEnumerableModelChoiceField(enum_forms.EnumerableModelChoiceField,
+                                        CreatorChoiceMixin):
```

Comment:
- It's a breaking change no (this class exists in creme 2.4) ? => Changelog
- Put the Mixin before the base field.

Reply:
I pushed a bit too fast, before saving the changelog. It is done now. Along the mixin change.